pynestml/cocos/co_co_only_spike_buffer_data_types.py

```diff
@@ -25,15 +25,15 @@
 
 class CoCoOnlySpikeBufferDataTypes(CoCo):
     """
-    This coco ensures that all spike buffers have a data type and all current buffers no data type stated.
+    This coco ensures that all spike and current buffers have a data type stated.
     Allowed:
         input:
-            spikeIn integer <- inhibitory spike
+            spikeIn pA <- inhibitory spike
```

Comment:
I would leave the previous example and add ``current pA <- current`` (or whatever we decide to name it), making it 2 lines inside of the input block. I would also leave spike as integer